classes/Device.class.php

```diff
@@ -1871,7 +1871,7 @@ function GetChildDevicePicture($parentDetails, $rear=false){
 						$slot->H=$ymax-$ymin;
 					}else{
 						// Last slot isn't defined so just error out
-						break;
+						return;
```

Comment:
Fair enough, which version of PHP are you running as well.  I hadn't noticed that error in my logs but I might not be triggering the right condition for it either.